src/content/app/tools/blast/components/listed-blast-submission/ListedBlastSubmission.scss

```diff
@@ -82,6 +82,7 @@ and also that it can be of variable width
 
 .jobStatus {
   justify-self: end;
+  width: 65px;
```

Comment:
What text can go in the jobStatus column? Are we quite sure that `width: 65px` is sufficient for all these text labels?